assets/js/renderer.js

```diff
@@ -423,6 +423,29 @@ $(document).ready(function () {
         });
     });
 
+    // Save a SQLite Database File -- custom menu
+    ipcRenderer.on('save-SQLite', function () {
+        dialog.showSaveDialog(
```

Comment:
@johnmhoran Could you refactor the duplicated code in the `ipcRenderer` and the `saveDBFileButton` click event into a separate function and have them both call that function?

You could also change the `saveDBFileButton` to `saveSQLiteFileButton` so that we are consistent.